extensions/proposals/WEBGL_texture_source_iframe/extension.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- vi:set sw=2 ts=4: -->
+<?xml-stylesheet href="../../extension.xsl" type="text/xsl"?>
+
+<proposal href="proposals/WEBGL_texture_source_iframe/">
+  <name>WEBGL_texture_source_iframe</name>
+  <contact><a href="https://www.khronos.org/webgl/public-mailing-list/">WebGL
+  working group</a> (public_webgl 'at' khronos.org) </contact>
+  <contributors>
+    <contributor>Zhenyao Mo, Google Inc</contributor>
+    <contributor>Members of the WebGL working group</contributor>
+  </contributors>
+  <number>NN</number>
+  <depends>
+    <api version="1.0"/>
+    <api version="2.0"/>
+  </depends>
+  <overview>
+    <p>This extension enables WebGL implementations to bind an HTMLIFrameElement object as the data source
+    to a texture. While bound, the extension provides an API to allow applications to request the latest
+    iframe rendering results to be blitted to the texture. The extension also provides an API to allow
+    applications to transform and forward related user input events from WebGL canvas to the bound iframe,
+    thus enabling the bound iframe to be interative inside a WebGL scene.</p>
+    <p>Due to security concerns, currently this extension only supports same origin iframes. This
+    limitaion may be lifted in the future.</p>
+  </overview>
+  <idl xml:space="preserve">
+[NoInterfaceObject]
+interface WEBGL_texture_source_iframe {
+  Promise bindTextureSource(GLenum target, HTMLIframeElement iframe);
+  Promise requestFrame(GLenum target);
+  void setEventForwarding(function(Event));
+};
+  </idl>
+  <newfun>
+    <function name="bindTextureSource" type="Promise">
+      <param name="target" type="GLenum"/>
+      <param name="iframe" type="HTMLIFrameElement"/>
+      <p>
+	Connect an iframe to the texture currently bound to <code>target</code>. Once the iframe
+	rendering is ready to be blitted to the texture, return a Promise. If <code>iframe</code>
```

Comment:
I thought we wanted to avoid blitting and get the browser to render directly into the GPU memory that will be sampled.

Reply:
I didn't mean for the spec to be specific about implementation.  If blitting is understood as one copy, then we should re-word it.  Chrome implemented it as one copy and is efficient enough.  Of course we will think about zero copy implementation, but the spec should not get into that.

Reply:
How about "once the iframe rendering is ready for sampling"?

I agree the spec should not be specific about implementation and yes blitting is understood as one copy.